src/Stratis.Features.SQLiteWalletRepository/SQLiteWalletRepository.cs

```diff
@@ -976,6 +985,7 @@ private bool StartBatch(ProcessBlocksInfo round, ChainedHeader header)
                 // Initialize round.
                 round.PrevTip = (header.Previous == null) ? new HashHeightPair(0, -1) : new HashHeightPair(header.Previous);
                 round.NewTip = null;
+                round.Trackers = new Dictionary<TopUpTracker, TopUpTracker>();
```

Comment:
This means that after each batch we reset the trackers yes?

Reply:
Yes.